content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -76,51 +76,78 @@ how to [authenticate apiservers](#authenticate-apiservers).
 ### Deploy the admission webhook service
 
 The webhook server in the e2e test is deployed in the Kubernetes cluster, via
-the [deployment API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#deployment-v1beta1-apps).
+the [deployment API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#deployment-v1-apps).
 The test also creates a [service](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#service-v1-core)
 as the front-end of the webhook server. See
-[code](https://github.com/kubernetes/kubernetes/blob/v1.13.0/test/e2e/apimachinery/webhook.go#L227).
+[code](https://github.com/kubernetes/kubernetes/blob/v1.15.0/test/e2e/apimachinery/webhook.go#L301).
 
 You may also deploy your webhooks outside of the cluster. You will need to update
-your [webhook client configurations](https://github.com/kubernetes/kubernetes/blob/v1.13.0/staging/src/k8s.io/api/admissionregistration/v1beta1/types.go#L247) accordingly.
+your webhook configurations accordingly.
 
 ### Configure admission webhooks on the fly
 
 You can dynamically configure what resources are subject to what admission
 webhooks via
-[ValidatingWebhookConfiguration](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#validatingwebhookconfiguration-v1beta1-admissionregistration-k8s-io)
+[ValidatingWebhookConfiguration](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#validatingwebhookconfiguration-v1-admissionregistration-k8s-io)
 or
-[MutatingWebhookConfiguration](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#mutatingwebhookconfiguration-v1beta1-admissionregistration-k8s-io).
+[MutatingWebhookConfiguration](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#mutatingwebhookconfiguration-v1-admissionregistration-k8s-io).
 
-The following is an example `validatingWebhookConfiguration`, a mutating webhook
-configuration is similar. See the [webhook configuration](#webhook-configuration) section for details about each config field.
+The following is an example `ValidatingWebhookConfiguration`, a mutating webhook configuration is similar.
+See the [webhook configuration](#webhook-configuration) section for details about each config field.
 
+{{< tabs name="ValidatingWebhookConfiguration_example_1" >}}
+{{% tab name="admissionregistration.k8s.io/v1" %}}
+```yaml
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: "pod-policy.example.com"
+webhooks:
+- name: "pod-policy.example.com"
```

Comment:
Is there a naming convention of configurations and the webhooks that they contain that we want to promote? I.e. do we want to encourage or discourage conversion name == webhook name for single webhook configurations?

I can imaging there might be some future proofing advantages of naming a configuration in a way that more generally covers the type of webhooks it might be responsible for, .e.g. security policy, or service mesh, and the webhooks might be named by the particular task they perform...

Apologies if this is out of scope for this PR, it just got me thinking..

Reply:
For extension point resources, I'd tend to encourage naming the API objects with self-regulating names (e.g. suffixed with a domain you control)